mlflow/tracking/fluent.py

```diff
@@ -206,6 +206,8 @@ def log_param(key, value):
     """
     Log a parameter under the current run, creating a run if necessary.
 
+    **Note:** If no run is active, it will instantiate a new run.
```

Comment:
Thanks for adding these - I see the docs for most of these functions already mention that we create a run, maybe we could update that wording to reflect that we create an "active" run instead of repeating that a run is created?

e.g. for this method we could say:

> Log a parameter under the current active run, creating a run if necessary.

Reply:
Done.